src/components/Footer.css

```diff
@@ -0,0 +1,15 @@
+.App-footer {
+  display: flex;
+  justify-content: space-around;
```

Comment:
as you only have 1 div in your footer, `justify-content: center` would fit better, I think.

Reply:
you are right ! It´s now changed :)